taxonomy/utils.py

```diff
@@ -25,25 +25,33 @@
 CACHE_TIMEOUT_COURSE_SKILLS_SECONDS = 60 * 60
 
 
-def get_whitelisted_serialized_skills(course_key):
+def get_whitelisted_serialized_skills(key_or_uuid, product_type):
```

Comment:
nit: We can default product type to 'Course'. We would not need to change it across many places then.